EthOn.rdf

```diff
@@ -1904,18 +1904,18 @@
         </owl:annotatedTarget>
         <rdfs:comment>All create transactions are directed toward a Contract Account they create.</rdfs:comment>
     </owl:Axiom>
-
+    
 
 
     <!-- http://ethon.consensys.net/EthOnConcept -->
 
     <owl:Class rdf:about="http://ethon.consensys.net/EthOnConcept">
         <suggestedStringRepresentation xml:lang="en">EthonConcept</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Groups al EthOn concepts.</rdfs:comment>
+        <rdfs:comment xml:lang="en">The purpose of concepts in EthOn is to group a finite number of concrete instances of states and/or relationships between states under a single indefinitely bounded term.</rdfs:comment>
```

Comment:
Perhaps this is an appropriate place to begin using the 'simpleDefinition' annotation. I think a version of @chronaeon 's definition, revised to be more concise without sacrificing too much in terms of ontological rigor, would be useful as the rdfs:comment attribute, while the definition @jo-tud suggested is sufficient for most casual users.

I will take a stab at this and share it for feedback as soon as possible.